.travis.yml

```diff
@@ -8,6 +8,6 @@ script:
 - bundle exec rspec
 deploy:
   provider: heroku
-  # api_key: 
-  app: https://website-resume-luke.herokuapp.com/
+  # api_key:
+  app: https://shopify-image-repo-lhje.herokuapp.com/
```

Comment:
All of what was on master (soon to be renamed to main) came from a general template that I've made that, it appears, we need to update to be a little more general.